css/pages/index.css

```diff
@@ -19,3 +55,20 @@
     font-size: 1rem;
   }
 }
+
+@media screen and (max-width: 430px) {
+  .container {
+    max-width: 350px;
+    font-size: 0.7em;
+  }
+  .card {
+    width: 5rem;
```

Comment:
What's `rem` doing in width 🤔?
Can we use % for responsiveness?

https://medium.com/dev-genius/a-bite-sized-best-practices-guide-for-css-units-em-rem-px-and-more-f621c0d56df4

Reply:
> What's `rem` doing in width 🤔?
> Can we use % for responsiveness?
> 
> https://medium.com/dev-genius/a-bite-sized-best-practices-guide-for-css-units-em-rem-px-and-more-f621c0d56df4

Oh yeah u r right I should use % for responsiveness